### PR TITLE
chore(monitoring): Record controller invocations.

### PR DESCRIPTION
**Pull request by duftler**

### CODE REVIEW

File: kayenta-web/src/main/java/com/netflix/kayenta/interceptors/MetricsInterceptor.java [hunk 1]
```diff
@@ -0,0 +1,139 @@
+/*
+ * Copyright 2015 Netflix, Inc.
```
Comment (duftler) [inline kayenta-web/src/main/java/com/netflix/kayenta/interceptors/MetricsInterceptor.java]: Left this as I simply copied [this](https://github.com/spinnaker/kork/blob/master/kork-web/src/main/java/com/netflix/spinnaker/kork/web/interceptors/MetricsInterceptor.java) and replaced the path variable lookup with a query param lookup.

**Comment by duftler** [comment]
Closes #360.

File: kayenta-web/src/main/java/com/netflix/kayenta/interceptors/MetricsInterceptor.java (hunk 1 above)
Comment (ewiseblatt) [reply kayenta-web/src/main/java/com/netflix/kayenta/interceptors/MetricsInterceptor.java]: mention this is copied from the kato url

Comment (duftler) [reply kayenta-web/src/main/java/com/netflix/kayenta/interceptors/MetricsInterceptor.java]: Done.

File: kayenta-web/src/main/java/com/netflix/kayenta/interceptors/MetricsInterceptor.java [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+      }
+
+      Id id = registry.createId(metricName)
+        .withTag("controller", controller)
+        .withTag("method", handlerMethod.getMethod().getName())
+        .withTag("status", status.toString().charAt(0) + "xx")
+        .withTag("statusCode", status.toString());
+
+      List<String> paramNames = Collections.list(request.getParameterNames());
+
```
Comment (ewiseblatt) [inline kayenta-web/src/main/java/com/netflix/kayenta/interceptors/MetricsInterceptor.java]: Can you mark this as the place that differs, and perhaps todo generalizing kork in such a way as to support this use case so you dont need this variant?

Leave for now but note it.

Comment (duftler) [reply kayenta-web/src/main/java/com/netflix/kayenta/interceptors/MetricsInterceptor.java]: Done.

**Pull request merged**